README.md

```diff
@@ -130,26 +130,31 @@ CommonMarker accepts the same options that CMark does, as symbols. Note that the
 | Name                          | Description
 | ----------------------------- | -----------
 | `:DEFAULT`                    | The default parsing system.
+| `:SOURCEPOS`                  | Include source position in nodes
```

Comment:
As far as I can tell, we check for `SOURCEPOS` in `inlines.c`, thereby allowing someone implementing their own renderer to also add sourpos info to their output.